remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/mapping/model/ProtobufRawTypeModel.java

```diff
@@ -13,6 +14,7 @@
 import org.hibernate.search.mapper.pojo.model.spi.PojoPropertyModel;
 import org.hibernate.search.mapper.pojo.model.spi.PojoRawTypeIdentifier;
 import org.hibernate.search.mapper.pojo.model.spi.PojoRawTypeModel;
+import org.hibernate.search.mapper.pojo.model.spi.PojoTypeModel;
 
 public class ProtobufRawTypeModel implements PojoRawTypeModel<byte[]> {
```

Comment:
Not strictly related to this PR, but `ProtobufRawTypeModel` could extend `AbstractPojoRawTypeModel`. That would allow you to get rid of some code.

Reply:
I tried but the resulting code is not great. Since we don't use introspector here, I would keep it as it is.